en/sim_gazebo_gz/gazebo_vehicles.md

```diff
@@ -0,0 +1,60 @@
+# Gazebo Vehicles
+
+This topic lists/displays the vehicles supported by the PX4 [Gazebo](../sim_gazebo_gz/README.md) simulation, and the `make` commands required to run them (the commands are run from a terminal in the **PX4-Autopilot** directory).
+
+Supported vehicle types include: mutirotor, VTOL, Plane.
+
+:::note
+The [Gazebo](../sim_gazebo_gz/README.md) page shows how to install Gazebo.
+:::
+
+## Multicopter
+
+### X500 Quadrotor
+
+```sh
+make px4_sitl gz_x500
+```
+
+### X500 Quadrotor with Visual Odometry
+
+```sh
+make px4_sitl gz_x500_vision
+```
+
+![x500 in Gazebo](../../assets/simulation/gazebo/vehicles/x500.png)
+
+### X500 Quadrotor with Depth Camera
+
+These models have a depth camera attached, modelled on the OAK-D.
+
+_Forward-facing depth camera:_
+
+```sh
+make px4_sitl gz_x500_depth
+```
+
+![x500 with depth camera in Gazebo](../../assets/simulation/gazebo/vehicles/x500_depth.png)
+
+<a id="fixed_wing"></a>
+## Plane/Fixed Wing
+
+<a id="rc-cessna"></a>
```

Comment:
FYI this is correct, though you could link to standard-plane if you wanted

Reply:
Links are "better" if you use the Vupress heading anchor - that always takes you to the right place. Manual anchors like this are always a little off where you want them to go. Sometimes I have used manual anchors because they stay even if the heading name changes - things don't get broken unexpectedly. Also if headings are very long so the link is too.

Anyway, this anchor is fine.